commands/psp-publish.js

```diff
@@ -317,10 +340,19 @@ async function readFilePromise (filePath) {
   });
 }
 
+async function getFileUpdatedTimePromise (filePath) {
```

Comment:
I think it okay to avoid suffixing async functions with `Promise`. The `await getFileUpdateTime()` form communicates that it is returning a Promise.

Reply:
right... this is to be consistent with the existing `readFilePromise` function.